tests/mockUtil.ts

```diff
@@ -41,6 +98,31 @@ export const testVscode: any = {
   ShellExecution: class MockShellExecution {},
   Task: class Task {},
   TaskScope: { Workspace: true },
+  languages: {
+    createDiagnosticCollection: (): DiagnosticCollection => {
+      return {
+        set: (
+          entries: ReadonlyArray<[Uri, ReadonlyArray<unknown> | undefined]>
+        ) => {
+          entries.toString();
```

Comment:
Why are you calling `toString`?

Reply:
because of eslint.
I can add
`eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
Please disable the eslint issue or remove the parameter definition